spring-integration-smb/src/main/java/org/springframework/integration/smb/session/SmbConfig.java

```diff
@@ -134,6 +147,50 @@ public boolean isUseTempFile() {
 		return this.useTempFile;
 	}
 
+	/**
+	 * Gets the desired minimum SMB version value for what the Windows server will allow
+	 * during protocol transport negotiation.
+	 *
```

Comment:
No blank lines in the method JavaDocs, please: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting

Reply:
Done! Thanks.